src/app/trips/trip-overview/trip-overview.component.html

```diff
@@ -9,12 +9,12 @@ <h1>{{trip.title}}</h1>
 </span>
 
 <div>
-  <p>Destination<span *ngIf="multipleLocations">s</span>:</p>
-  <ng-container *ngFor="let location of trip?.locations">
+  <p>Destination<span *ngIf="multipleDestinations">s</span>:</p>
```

Comment:
It looks like the changes in this file are used in a few locations. This is a good opportunity to create a component to state one time what it looks like to display a list of destinations. Then you can use that list of destinations anywhere and keep a consistent look. it would be something like this

```html
<app-destination-list [destinations]=destinations></app-destination-list>
```